app/MainHie.hs

```diff
@@ -111,22 +111,13 @@ run opts = do
 
   Core.setupLogger mLogFileName ["hie", "hie-bios"] logLevel
 
-  d <- getCurrentDirectory
-  -- Get the cabal directory from the cradle
-  cradle <- findLocalCradle (d </> "File.hs")
-
-  projGhcVersion <- getProjectGhcVersion cradle
```

Comment:
Does this remove the GHC version check?

Reply:
There is an identical check in LspStdio.hs, so it will only remove it if the user selects the JSON transport.

Reply:
We need to remove the JSON transport, pretty sure it is unused. Particularly as the build plugin has been removed already, which is the only thing being used via that, to my knowledge.

Reply:
Is that something that should be done in hie-bios pr? I volunteer to delete code.

Reply:
Maybe we should do it after as a standalone PR, unless it is causing a problem for the hie-bios PR.

Reply:
No, it does not. I agree, we can remove it after merge.